playbooks/services.yml

```diff
@@ -1,3 +1,8 @@
+# https://github.com/scality/metal-k8s/issues/120
+- hosts: k8s-cluster:etcd
+  tasks:
+    - debug: msg="Ensure facts are gathered"
```

Comment:
the tasks: "setup"  (with gather_fatcs: False)

Reply:
What do you mean exactly :question:

Reply:
oups forget the word "or". For gathering facts ansible just execute the special tasks "setup" that you can use instead of debug task (but don't forget to set gather_facts: False, unless you want ansible to gather facts twice). Just to let you know. It's perfectly understandable in your version, which is all I care.